webapp/channels/src/components/admin_console/license_settings/license_settings.test.tsx

```diff
@@ -181,7 +181,7 @@ describe('components/admin_console/license_settings/LicenseSettings', () => {
         const instance = wrapper.instance();
 
         // First start trial
-        actions.requestTrialLicense = jest.fn().mockImplementation(() => Promise.resolve({percentage: 1, error: null}));
+        actions.requestTrialLicense = jest.fn().mockImplementation(() => ({percentage: 1, error: null}));
```

Comment:
is this change intended?

Reply:
Not really, probably part of the multiple conflicts I've been solving. Removed.